les/vflux/server/clientpool.go

```diff
@@ -239,12 +239,11 @@ func (cp *ClientPool) SetCapacity(node *enode.Node, reqCap uint64, bias time.Dur
 			maxTarget = curve.maxCapacity(func(capacity uint64) int64 {
 				return balance.estimatePriority(capacity, 0, 0, bias, false)
 			})
-			if maxTarget <= capacity {
+			if maxTarget < reqCap {
```

Comment:
Actually this is not really correct, with this change `SetCapacity` cannot do a partial capacity increase. `maxTarget` is the highest available capacity according to the curve based estimation. If it's somewhere between current `capacity` and `reqCap` then we should not exit, we should still attempt `requestCapacity`.
Still, since we don't use partial capacity increase yet (`SetCapacity` is only accessible from the API right now), you can merge this PR as it is if the fuzzer error is annoying and I'll figure out a really robust method later.

Reply:
Hmm. I don't know. Perhaps a simple solution for this scenario is that the server can respond the current available capacity to the client and client needs to re-request the capacity with a lower target.

But let's see how will this partial capacity work.